src/assets/i18n/en_US.json

```diff
@@ -798,6 +798,7 @@
   "checkout.order_review.heading.title": "Review your order information",
   "checkout.order_review.send.button": "Submit order",
   "checkout.order_summary.heading": "Order summary",
+  "checkout.payment.addPayment.form.legend": "Payment parameter form for {{paymentMethodName}}",
```

Comment:
Should this be "Payment instrument parameter form" here, as in the key below? Please also see my comments at the German localization.